### PR TITLE
fix staticroute router

### PR DESCRIPTION
**Pull request by dpfaffenbauer**
seems that part from this PR https://github.com/pimcore/pimcore/pull/5946 got removed with this PR https://github.com/pimcore/pimcore/pull/6096 by accident

### CODE REVIEW

**Comment by dpfaffenbauer** [comment]
Test fail is not related to this PR

File: lib/Routing/Staticroute/Router.php [hunk 1]
```diff
@@ -236,14 +237,22 @@ public function match($pathinfo)
      *
```
Comment (NiklasBr) [inline lib/Routing/Staticroute/Router.php:237]:
```suggestion
     * @param Request|null $request
```

**Comment by brusch** [comment]
Thanks a lot! 🙏

**Comment by brusch** [comment]
Accidentally merged that to the master an not to `6.6` therefore cherry picked that here 95c0801db86dc84a57e6a6ce2daaf675bd79af78

**Pull request merged**